sklearn/utils/_testing.py

```diff
@@ -760,6 +776,10 @@ def assert_docstring_consistency(
         List of returns to be excluded. If None, no returns are excluded.
         Can only be set if `include_returns` is True.
 
+    description_regex : str, default=""
+        Regular expression to match to all descriptions. If empty string, will
```

Comment:
In this regard, I would rephrase it slightly:

```
Regular expression pattern to match all descriptions.
```

Maybe we should be more explicit what we mean by "descriptions".